reloj_checador_team03/app/views/users/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<%= toastr_flash %>
+<h1>Users</h1>
+<div class="container-content">
+
+  <div class="d-flex justify-content-between bd-highlightm mb-3">
+    <div class="d-flex justify-content-start">
+     <%= form_tag(users_search_path, method: "post", remote: true, authenticity_token: true, role: "form",class: "form") do %>
+     <%= text_field_tag('search', nil, placeholder: 'Enter search term...', class: 'form-control') %>
+     <% end %>
+    </div>
+
+    <div class="d-flex justify-content-center">
+      <a class="btn btn-success create-button" id="new-user-btn">New User</a>
+    </div>
+  </div>
+
+  <div class="modal fade" id="user-modal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+   
+  </div>
+
+  <table class="table">
+    <thead class="table-thead">
+       <tr>
+        <th class="table-thead-th">Name</th>
+        <th class="table-thead-th">Lastname</th>
+        <th class="table-thead-th">Username</th>
+        <th class="table-thead-th">Email</th>
+        <th class="table-thead-th">Actions</th>
+      </tr>
+    </thead>
+
+    <tbody id="user-table-body">
+     <%= render 'search', remote: true %>
+    </tbody>
+  </table>
+   <%= paginate @users %>
+</div>
+<script>
+$('#new-user-btn').click(function(){
+
+  $("#user-modal").html("<%= j render '/users/registrations/new' %>");
+  $('#user-modal').modal('show');
+})
+</script>
```

Comment:
Personally, I recommend using specific files for JS, to avoid mixing HTML and JS code.